src/core/overlay/overlay.view.js

```diff
@@ -0,0 +1,52 @@
+import View from 'core/view/view';
+
+export default class OverlayView extends View {
+	constructor(model, markup){
+		super();
+
+		this.model = model;
+		this.markup = markup;
+
+		this.onInit();
+	}
+
+	onInit() {
+		let overlayElement = this.markup.document.querySelector('q-grid-range-overlay');
```

Comment:
this.markup.document.querySelector('q-grid-range-overlay') this should go from markup
e.g. this,markup.range